pkg/iris/workloads/services/mapper/mapping.go

```diff
@@ -1,15 +1,19 @@
 package mapper
 
 import (
-	"github.com/leanix/leanix-k8s-connector/pkg/iris/workloads/models"
+	workload "github.com/leanix/leanix-k8s-connector/pkg/iris/workloads/models"
 	"github.com/leanix/leanix-k8s-connector/pkg/kubernetes"
+	"github.com/leanix/leanix-k8s-connector/pkg/set"
+	v1 "k8s.io/api/core/v1"
+	"strings"
 )
 
 type MapperWorkload interface {
-	MapWorkloads(clusterName string) ([]models.Workload, error)
+	MapCluster(clusterName string, nodes *v1.NodeList) (workload.Cluster, error)
+	MapWorkloads(cluster workload.Cluster) ([]workload.Data, error)
 }
 
-type mapworkload struct {
+type MappedWorkload struct {
```

Comment:
The cleanest way would be to:
- rename MapperWorkload interface to WorkloadMapper ([effective go](https://go.dev/doc/effective_go#interface-names))
- rename the struct to workloadMapper (lowercase since the access could be private right?)
At least I would rename MappedWorkload to mapperWorkload